### ISSUE TITLE
Fix from stdin writes more than the fixed sql to stdout

### ISSUE THREAD

**Issue by nolanbconaway** [alanmcruickshank/sqlfluff#323]
When it was originally written, fixing from stdin printed the fixed sql to stdout and noting else. That way you could do things like 

```sh
echo ' select * from t' | sqlfluff fix - --rules L003 > fixed.sql
```
At least in 0.3.3, a lint result gets printed out, and the fixed string is printed as a python tuple:

```sh
bash-3.2$ sqlfluff version
0.3.3
bash-3.2$ echo ' select * from t' | sqlfluff fix - --rules L003
== [stdin] FAIL
L:   1 | P:   2 | L003 | First line has unexpected indent
('select * from t\n', True)
```

**Comment by nolanbconaway** [comment]
@alanmcruickshank i am happy to debug this and submit a PR, simply want to confirm that this is not the desired behavior!

**Comment by alanmcruickshank** [comment]
I think this is an unexpected side effect of another change. We should probably suppress that output when running from stdin. If you're happy to put together a PR that would be awesome. You'll notice that the return value of some functions has changed since our original PR too.

Could you also put in place a test case for this so that we can catch if it happens again in future?

Thanks!

**Comment by nolanbconaway** [comment]
Yeah definitely, on the case!
